baselines/Gmail Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -877,10 +864,10 @@ This section determines whether users are prompted with a warning for messages t
 ### Policies
 
 #### GWS.GMAIL.13.1v0.2
-Unintended external reply warnings SHALL be enabled to avoid unintentional replies and remind users to treat external messages with caution.
+Unintended external reply warnings SHALL be enabled
 
-- _Rationale:_ Unintended external reply warnings can help reduce the risk of exposing sensitive information in replies to external messages. Enabling these warnings reminds users to treat external messages with caution, reducing this risk and enhancing the safety and integrity of user data and systems.
-- _Last modified:_ July, 10, 2023
+- _Rationale:_ Unintended external reply warnings can help reduce the risk of exposing sensitive information in replies to external messages. Enabling these warnings reminds users to treat external messages with caution, reducing this risk and enhancing the safety and integrity of user data and systems. This helps avoid unintentional replies and remind users to treat external messages with caution.
```

Comment:
The rationale already says that this reminds "reminds users to treat external messages with caution," no need to state it twice.

```suggestion
- _Rationale:_ Unintended external reply warnings can help reduce the risk of exposing sensitive information in replies to external messages. Enabling these warnings reminds users to treat external messages with caution, reducing this risk and enhancing the safety and integrity of user data and systems. Additionally, this may help avoid unintentional replies.
```